project/Dependencies.scala

```diff
@@ -57,7 +61,7 @@ object Dependencies {
 
     // Excluding grpc-alts works around a complex resolution bug
     // Details are in https://github.com/akka/akka-grpc/pull/469
-    val grpcInteropTesting = ("io.grpc" % "grpc-interop-testing" % Versions.grpc)
+    val grpcInteropTesting = ("io.grpc" % "grpc-interop-testing" % Versions.grpcInterop)
```

Comment:
Still some tests failing because of this. I was somewhat worried that this would be a showstopper for updating, but I have a tried a snapshot with various gprc projects and it seems to work fine.